workflows/bacterial_genomics/bacterial_genome_assembly/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [0.1] - 05-06-2024
```

Comment:
This doesn't match the release (1.0) stated in the WF file!

Reply:
Thanks !